xnat17/xnat.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+OWD=$PWD
+
+
+mkdir -p /data/xnat/home/config 2> /dev/null
+cd /data/xnat/home
+mkdir -p logs plugins work 2> /dev/null
```

Comment:
It would probably be better to use absolute paths to create these directories. You would not need to `cd` around, and would not need to store the `OWD`, and all that. It would just be cleaner to make the directories and not have to move around.